content/en/docs/concepts/storage/volumes.md

```diff
@@ -1017,6 +1017,54 @@ spec:
         claimName: my-lamp-site-data
 ```
 
+### Using subPath with expanded environment variables
+
+{{< feature-state for_k8s_version="v1.11" state="alpha" >}}
+
+{{< note >}}
+**Note:** The feature gate VolumeSubpathEnvExpansion must be enabled
+{{< /note >}}
+
+`subPath` directory names can also be constructed from Downward API environment variables. Here is an example of a Pod that uses `subPath` to create a logs directory with the pod name.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: env-image
+spec:
+  containers:
+  - name: env-container
+    env:
+    - name: POD_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.name
+    image: busybox
+    command: [ "sh", "-c", "while [ true ]; do echo 'Hello'; sleep 10; done | tee -a /logs/hello.txt" ]
+    volumeMounts:
+    - name: workdir1
+      mountPath: /logs
+      subPath: $(POD_NAME)
+  restartPolicy: Never
+  volumes:
+  - name: workdir1
+    hostPath: 
+      path: /logs
+```
+
+To check that the output is correct, the following will appear on the container mounted `hostPath`
+
+```shell
+cat /logs/env-image/hello.txt
```

Comment:
You can't actually access this path from the container since only the subpath is mounted in.

So either this example needs to also mount the hostpath into this or another container.  Or add a line saying that this is what the directory looks like from the host.

Reply:
Corrected ambiguous statement